universal-login-relayer/test/integration/core/services/MessageHandler.js

```diff
@@ -30,7 +31,7 @@ describe('INT: MessageHandler', async () => {
   });
 
   it('Error when not enough tokens', async () => {
-    const message = {...msg, gasLimitExecution: utils.parseEther('2.0'), gasData: 8912};
+    const message = {...msg, gasLimitExecution: utils.parseEther('2.0'), gasData: 8976};
```

Comment:
#messageToSignedMessage

Reply:
Do we still need `gasData` here?

Reply:
Can we get rid of `gasData` and `createSignedMessage` at all in this file?

Reply:
I'd like to discuss token payment tomorrow

Reply:
Ok